data/options.yaml

```
@@ -28,9 +28,9 @@
 name: Player{number}
 
 # Used to describe your yaml. Useful if you have multiple files.
-description: Default {{ game }} Template
+description: {{ yaml_dump("Default " + game + " Template") }}
```

I think we almost always prefer string formatting over string concatenation. I believe jinja can't do f-strings, so this would be the "proper" way.

```suggestion
description: {{ yaml_dump("Default %s Template" % game) }}
```

---

Thanks for the suggestion!